src/commands/api-mesh/__tests__/describe.test.js

```diff
@@ -174,6 +175,11 @@ describe('describe command tests', () => {
 		    "Mesh ID: %s",
 		    "dummy_meshId",
 		  ],
+		  [
+		    "Mesh Endpoint: %s
+		",
+		    "https://graph.adobe.io/api/dummy_meshId/graphql",
```

Comment:
Shouldn't this non-TI mesh endpoint need to have the "dummy_apiKey" ?

Reply:
Actually this test is for verifying that if `api_key` is missing then test should not fail, it's name is also `should not fail if api key is missing from mesh details`.

So as `api_key` is not given in `toMatchInlineSnapshot` we get empty `api_key` and as a result there is no `api_key` in returned url.

Just below this test there is a test in which we are passing `api_key` in `toMatchInlineSnapshot`, so we get `api_key` and it is returned in url as well and we are also asserting against url that contains `api_key`, [here](https://github.com/adobe/aio-cli-plugin-api-mesh/blob/ce5311638e5b1f7147e1a9dc75b2662993966942/src/commands/api-mesh/__tests__/describe.test.js#L229)